src/components/Checkbox/index.ts

```diff
@@ -1 +1 @@
-export { Checkbox } from './Checkbox';
+export { Checkbox, Props as CheckboxProps } from './Checkbox';
```

Comment:
What about exporting from `packages`, `utils`, `constants`?

Reply:
I've added export for constants, packages have been already exported and utils (default) about others utils i don't think we need to export them

Reply:
what about the types in `types/types.d.ts`?

Reply:
I think we don't need it, also it's not a module

Reply:
It is used in our components. In applications using OneUI, TS cannot validate these props, as the definition is not exported, and TS doesn't know what they mean.

Reply:
@mukiienko @eszthoff Is it will be okay if after add export for types/types.d.ts that we need to import  types to components in OneUI?

Reply:
I think it should be a way to expose `d.ts files in the build via Webpack.

Reply:
@mukiienko as i understand the idea is that we can see types here
<img width="1048" alt="Screenshot 2022-12-19 at 13 43 30" src="https://user-images.githubusercontent.com/104063210/208682632-26328d9f-375e-47d9-adbf-9982c2ff7823.png">

Reply:
@mukiienko @eszthoff 
I've added solution with copy-webpack-plugin ( https://webpack.js.org/plugins/copy-webpack-plugin/).  Is it wiil be okay to use ?
I also have another solution migrate types.d.ts to  .ts file and add importing in files where we use it

Reply:
I tried to build locally and had a quick look at the dist files. I still see that these types are not resolved there. E.g. `dist/dts/src/components/BulkActionsToolbar/BulkActionsToolbar.d.ts` have red highlights

<img width="729" alt="Screenshot 2022-12-27 at 14 45 42" src="https://user-images.githubusercontent.com/23654472/209675679-f027cb41-6fdd-4d49-8408-dc8fa4e42e37.png">